### PR TITLE
c26 translation update

### PR DESCRIPTION
**Pull request by ManlyMarco**
Thanks to @ScrewThisNoise for providing the translations.

### CODE REVIEW

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/00/84/translation.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「……コイカツって、何をやるの？」=“...Koikatsu, what is it anyway?”
+「……コイカツって、何をやるの？」=“...What are love activities, anyway?”
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/00/84/translation.txt:4]: Even if it wasn't the game title, it uses katakana, so it makes more sense to just use `Koikatsu`. Besides the next few lines explain it.
```suggestion
「……コイカツって、何をやるの？」=“...What is Koikatsu, anyway?”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/25/84/translation.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「……コイカツって、何をやるの？」=“...Koikatsu, what is it anyway?”
+「……コイカツって、何をやるの？」=“...What are love activities, anyway?”
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/25/84/translation.txt:4]: same as above

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/30/17/translation.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 //
 // Dumped for Koikatsu v5.1 (Koikatsu Party) by KKP_TextDump v1.4.6.0
 //
-「んーっ、よく寝た。……あれ？　メモがある。なになに――」=“Mmm, I slept well. ...Oh? There's a letter. What's it say...?”
-『おはよう。[P]、起きなかったから先に行ってるよ』="Good morning, [P]. I went ahead because you didn't wake up."
-『念のため、遅刻には気をつけて』="Just be careful that you aren't late."
-『P.S.また学園で、お話聞かせてね』="P.S. Talk with me at the school."
+「んーっ、よく寝た。……あれ？　メモがある。なになに――」=“Nnn, I slept well. ...Huh? There's a note. Let's see----”
+『おはよう。[P]、起きなかったから先に行ってるよ』=「Morning. You weren't waking up, so I'm leaving first.」
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/30/17/translation.txt:5]: Wrong quotes

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/30/84/translation.txt [hunk 4]
```diff
@@ -1,9 +1,9 @@
 //
 // Dumped for Koikatsu v5.1 (Koikatsu Party) by KKP_TextDump v1.4.6.0
 //
-「……コイカツって、何をやるの？」=“...The Love Hunter's Club?”
-「えっと、ざっくり言えば恋愛の研究……ってところかな」=“The study of romance, huh... I'm interested.”
-「……私、鯉に勝つ部活だと思ってた」=“...I thought it was just some kind of joke at first.”
-「そ、それはそれで興味があるような……」=“It sounds unique, at least...”
-「でも恋愛研究も……悪くない、かも。よろしくね」=“But it's not bad at least... Sure, where can I sign up?”
-「うん、よろしく。それじゃあ、今日は……」=“Yeah, thanks. Well then, here's what's going on today...”
+「……コイカツって、何をやるの？」=“...What are love activities, anyway?”
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/30/84/translation.txt:4]: same as above again

**Review by GeBo1 — CHANGES_REQUESTED** [review]

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/25/84/translation.txt (hunk 2 above)
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/25/84/translation.txt:4]:
```suggestion
「……コイカツって、何をやるの？」=“...What is Koikatsu, anyway?”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/30/84/translation.txt (hunk 4 above)
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/30/84/translation.txt:4]:
```suggestion
「……コイカツって、何をやるの？」=“...What is Koikatsu, anyway?”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/30/17/translation.txt (hunk 3 above)
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c26/30/17/translation.txt:5]:
```suggestion
『おはよう。[P]、起きなかったから先に行ってるよ』=“Morning. You weren't waking up, so I'm leaving first.”
```

**Review by GeBo1 — APPROVED** [review]

**Pull request merged**